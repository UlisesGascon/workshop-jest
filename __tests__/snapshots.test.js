// see: https://jestjs.io/docs/en/snapshot-testing

const data = [{
  name: 'Ulises',
  role: 'Developer'
}, {
  name: 'Peter',
  role: 'Developer'
}]

const dataTransformation = users => users.map(user => `- ${user.name} is ${user.role} in this team.`).join('')

describe('snapshot', () => {
  test('Simple data snapshot', () => {
    const dataTransformed = dataTransformation(data)
    expect(dataTransformed).toMatchSnapshot()
  })
})
