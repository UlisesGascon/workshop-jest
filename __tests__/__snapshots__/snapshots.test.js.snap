// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Simple data snapshot 1`] = `"- Ulises is Developer in this team.- Peter is Developer in this team."`;
